README.md

```diff
@@ -1,4 +1,8 @@
-# `rocks.nvim` (WIP)
+[<img src="https://raw.githubusercontent.com/luarocks/luarocks-logos/master/luarocks_logo_only.png" align="right" width="144" />](https://luarocks.org/)
+<img src="https://pngfre.com/wp-content/uploads/heart-87-1024x849.png" align="right" width="104" />
+[<img src="https://neovim.io/logos/neovim-mark-flat.png" align="right" width="114" />](https://neovim.io/)
+
+# `rocks.nvim`
```

Comment:
suggestion: Personally, I like to use HTML for readme headings (see for example the [luarocks-tag-release README](https://github.com/nvim-neorocks/luarocks-tag-release), and to spell out the plugin name as an SVG image (Lato font - same as Neovim uses), with "rocks" slim and ".nvim" bold.

(but that's just personal taste). If you like, I can play around with the image in Inkscape.

Reply:
Sounds good, I like the idea. I would definitely like to see it :)